src/java/org/apache/cassandra/hints/HintsStore.java

```diff
@@ -100,6 +101,22 @@ InetAddressAndPort address()
         return StorageService.instance.getEndpointForHostId(hostId);
     }
 
+    PendingHintsInfo getPendingHintsInfo()
```

Comment:
it would be nice to add a corresponding unit test in `HintsStoreTest`.

Reply:
Added in https://github.com/apache/cassandra/pull/1232/commits/b6877a5636a702fc045dfd8e1e4bbcf77ee8b959.